### ISSUE TITLE
Invalid version number

### ISSUE THREAD

**Issue by fredj** [camptocamp/ngeo#1101]
With nodejs 4.4.3, I get the following error on `npm install`:
```
npm ERR! install Couldn't read dependencies
npm ERR! Linux 4.4.0-21-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! node v4.4.3
npm ERR! npm  v2.15.1

npm ERR! Invalid version: "2.1.0.dev3"
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /home/fredj/code/ngeo/npm-debug.log
```

**Comment by fredj** [comment]
The number must be `2.1.0-dev3` instead of `2.1.0.dev3`.

See https://github.com/npm/node-semver#prerelease-tags

**Comment by fredj** [comment]
Fixed with 3b91f22877d838e1e4429b6b8cb7f6d4320aac5a
